lib/rubocop/cop/style/lambda.rb

```diff
@@ -62,8 +62,14 @@ def autocorrect(node)
 
         def autocorrect_new_to_old(corrector, node)
           block_method, args = *node
+          if add_whitespace?(block_method, args, node)
+            corrector.insert_before(node.loc.begin, ' ')
+          end
           corrector.replace(block_method.loc.expression, 'lambda')
-          return if args.children.empty?
+          if args.children.empty?
```

Comment:
This `if` looks kind of ugly because of the next `if` and the `return`. I'd suggest simplifying it a bit.